src/components/molecules/MenuPopover/MenuPopover.tsx

```diff
@@ -16,6 +20,9 @@ export const MenuPopover = ({ currentPath }: { currentPath: string }) => {
                             data-cy='popover-button'
                             className='flex flex-row items-center gap-1 whitespace-nowrap text-3.5 leading-4 text-neutral-800 outline-none'
                         >
+                            {isInMoreLinks && (
+                                <span className='mr-1 h-1.5 w-1.5 rounded-full bg-primary-500' />
```

Comment:
Could you fix this to follow the Nav style?
Also, the `More` text is moved to the right a little when it is active now but it shouldn't.